### PR TITLE
Exam_API Petriv: Compare currency rates on UI and API bdd test

### PR DESCRIPTION
**Pull request by yuliiapetriv**

### CODE REVIEW

File: PetrivProj/src/main/java/pages/PrivatBankMainPage.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import org.junit.Assert;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class PrivatBankMainPage extends ParentPage {
+
+    private final String PRIVAT_BANK_BASE_URL_UI = "https://privatbank.ua/";
+
+    @FindBy(xpath = ".//td[@id='USD_buy']")
```
[the remark this answers is not in the document]
Comment (yuliiapetriv) [reply PetrivProj/src/main/java/pages/PrivatBankMainPage.java]: fixed

File: PetrivProj/src/main/java/pages/PrivatBankMainPage.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    @FindBy(xpath = ".//td[@id='EUR_sell']")
+    private WebElement eurSaleRate;
+
+    public PrivatBankMainPage(WebDriver webDriver) {
+        super(webDriver);
+    }
+
+    @Override
+    protected String getRelativeUrl() {
+        return null;
```
[the remark this answers is not in the document]
Comment (yuliiapetriv) [reply PetrivProj/src/main/java/pages/PrivatBankMainPage.java]: fixed

File: PetrivProj/src/main/java/pages/PrivatBankMainPage.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+            logger.error("Can not open Main page");
+            Assert.fail("Can not open Main page");
+        }
+    }
+
+    @Step
+    public void storeRateForCurrency(String currencyName) {
+        switch (currencyName.toUpperCase()) {
+            case "USD":
+                TestData.USD_BUY_RATE_UI = Double.valueOf(usdBuyRate.getText());
```
[the remark this answers is not in the document]
Comment (yuliiapetriv) [reply PetrivProj/src/main/java/pages/PrivatBankMainPage.java]: fixed

File: PetrivProj/src/main/java/data/TestData.java [hunk 4]
```diff
@@ -13,4 +13,9 @@ public class TestData {
     public final static String VALID_LOGIN_DEMO_QA_API = "yuliiapetriv";
     public final static String VALID_PASSWORD_DEMO_QA_API = "!123Qwerty";
 
+    public static Double USD_BUY_RATE_UI = 0.0;
```
[the remark this answers is not in the document]
Comment (yuliiapetriv) [reply PetrivProj/src/main/java/data/TestData.java]: fixed

**Review by SeleniumQALight — APPROVED** [review]

**Pull request merged**